test/environment_test.rb

```diff
@@ -118,6 +118,32 @@
     end
   end
 
+  describe '#environment?' do
+    describe 'environment?(:test)' do
```

Comment:
What do you think about removing this line and change the 2 context description line to `when environment is matched` and `when environment is not matched` instead?